src/ncp-spinel/SpinelNCPInstance-DataPump.cpp

```diff
@@ -538,6 +538,16 @@ SpinelNCPInstance::driver_to_ncp_pump()
 		mOutboundBufferSent += pt->byte_count;
 #endif
 
+#ifdef USB_CDC_ACM_RESET
```

Comment:
Can we consider using`#if` check instead of `#ifdef`? 

I think in general we use (and prefer) `#if` style check in `wpantund` (allows all build configs to be defined in a common header file and the ones not needed to be set to zero).